src/components/ConversationSummary.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div class="blip-container summary">
+    <div :class="'bubble ' + position">
+      <div v-if="document.isLoading">
+        <div class="typing">
+          <div class="typing-dot"></div>
+          <div class="typing-dot"></div>
+          <div class="typing-dot"></div>
+        </div>
+      </div>
+      <div v-else>
+        <div class="title-summary">
+          <div class="title-icon">
+            <bds-icon
+              theme="outline"
+              name="robot"
+              style="margin-right: 8px;"
+              size="x-large"
+            />
+          </div>
+          <div class="flex justify-between title-text">
+            <bds-typo
+              variant="fs-16"
+              bold="bold"
+              tag="h1"
+              style="margin: 4px 8px 0px 0px"
+            >
+              {{ this.translations.titleConversationSummary }}
+            </bds-typo>
+            <bds-chip-tag color="outline">
+              Ticket #{{
+                `${this.document.sequentialId}${
+                  this.document.sequentialSuffix
+                    ? this.document.sequentialSuffix
+                    : ''
+                }`
+              }}
+            </bds-chip-tag>
+          </div>
+        </div>
+        <div v-if="status !== 'failed'" class="itens-summary">
+          <ul>
+            <li
+              v-for="(item, index) in this.document.summary"
+              v-bind:key="index"
+            >
+              <bds-typo variant="fs-16" bold="regular">{{
+                sanitizeText(item)
+              }}</bds-typo>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <blip-card-date
+      :status="status"
+      :position="position"
+      :date="date"
+      :failed-to-send-msg="failedToSendMsg"
+    />
+  </div>
+</template>
+
+<script>
+import { default as base } from '../mixins/baseComponent.js'
+
+export default {
+  name: 'conversation-summary',
+  mixins: [base],
+  props: {
+    document: {
+      type: Object,
+      required: true
+    },
+    translations: {
+      type: Object,
+      default: () => ({})
+    },
+    status: {
+      type: String,
+      default: ''
+    },
+    length: {
+      type: Number,
+      default: 532
+    },
+    disableLink: {
+      type: Boolean,
+      default: false
+    },
+    showMoreMsg: {
+      type: String,
+      default: 'Ver mais'
+    },
+    failedToSendMsg: {
+      type: String,
+      default: 'Falha ao enviar a mensagem'
+    }
```

Comment:
What is the difference between this `failedToSendMsg` and `failedToLoadConversationSummary` used in the `BlipGroupCard`?

Reply:
The message content is different
failedToSendMsg  --> Falha ao enviar a mensagem.
failedToLoadConversationSummary  --> Falha ao carregar o resumo da conversa.